generate/seccomp/consts.go

```diff
@@ -0,0 +1,7 @@
+package seccomp
+
+const (
+	overwrite = "overwrite"
+	appnd     = "append"
```

Comment:
On Thu, Jul 21, 2016 at 11:48:02AM -0700, Grant Seltzer Richman wrote:
> > @@ -0,0 +1,7 @@
> > +package seccomp
> > +
> > +const (
> > +	overwrite = "overwrite"
> > +	appnd     = "append"
>
> Well, the only issue is that it would then be exported, not sure if
> that makes a difference

If these are just for internal use, maybe prefix them (seccomp_append,
seccomp_overwrite).